ingest_revisions_dataflow/ingest_utils/test_ingester.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
```

Comment:
Add a comment at the top of this file... Is this file supposed to be a test? If so, better to name is ingester_test.py & also to add some actual validity test, e.g. check the the number of output lines is the number expected, and that some of the values are correctly pulled out?

Reply:
Done.